### DISCUSSION TITLE
Geo data seems be reverse.

### DISCUSSION THREAD

**Discussion by patrik-pa4k** [transitland/transitland#337] [Q&A] [answered]
Hello, when I'm trying to get data base on Lat and Lot, I'm providing the geo location of Bratislava ( Lat : 48.148596, Lon: 17.107748) and I receive empty results. When I send request with Lat : 17.107748, Lon: 48.148596 I receive response for Bratislava what is incorrect. I've tested also for Prague and seems to be same problem.
I'm providing some attachments as well from testing.

![image](https://user-images.githubusercontent.com/74901276/130452792-dab1ec88-150a-4071-bca9-aa4c9eac5c77.png)
![image](https://user-images.githubusercontent.com/74901276/130453984-b4a43b8c-4af1-4d70-8eac-b4d990333c9f.png)
![image](https://user-images.githubusercontent.com/74901276/130454216-86725222-3314-44f2-8889-20875565ee11.png)

**Comment by irees** [comment, marked as answer]
hi @patrik-pa4k 

Thank you for bringing this issue to my attention. You are correct, there was a coordinate reversal in a call to a postgis function. A fix is now deployed.

https://transit.land/api/v2/rest/stops.geojson?apikey=yourapikey&lon=17.1077&lat=48.148&radius=10000

<img width="551" alt="Screen Shot 2021-08-27 at 12 28 18 PM" src="https://user-images.githubusercontent.com/720087/131179148-2feacf0a-1771-44c0-b82f-94af45028b50.png">
